.github/workflows/pr_checklist.yml

```diff
@@ -14,22 +14,29 @@ jobs:
         with:
           script: |
             const comment = `
-            Thanks for creating a pull request! The below questions and checklist are intended to help with verifying your change is well tested. Response is optional, but if you choose to respond please edit this comment.
+            Thanks for creating a pull request! The below questions and checklist are intended to help with verifying your change is well tested. Please edit this comment by answering the questions below, or replacing the questions with your own description. Please remove this paragraph once you finish editing this comment.
+
+            ## Type of change made in this PR
+            - [ ] Functional change
+            - [ ] Test-only change
+            - [ ] Refactor-only change
+            - [ ] Other non-functional change
 
             ## What makes this change safe?
```

Comment:
I had a note about how `What makes this change safe?` encourages developers to argue a change is lower risk, but we should be encouraging them to call out when a change is medium or high risk. 
We say `How risky is this change?` below but I think we should also say this in the title.

Reply:
I believe the intention here is to use the sub-questions of risk evaluation, test coverage, and concerns collectively to provide the author and reviewers confidence that the change is safe to merge, rather than emphasizing one (e.g. risk evaluation) over others in the title. I also get your point the current title can be received as an "encouragement" to lower mark the risk the change deserves. 

Hope to hear other reviewers' opinions on this.

Reply:
Looks like there's no other opinion shared. I'd say let's leave it as is for now, and come back revisit it later if needed.